registrations/models.py

```diff
@@ -100,65 +158,87 @@ class NotificationType:
     registration = models.ForeignKey(
         Registration, on_delete=models.CASCADE, related_name="signups"
     )
-    name = models.CharField(verbose_name=_("Name"), max_length=50)
+
+    name = models.CharField(
+        verbose_name=_("Name"), max_length=50, blank=True, null=True, default=None
```

Comment:
I guess that this field might be sent as {"name": null} which is the reason for having `null=True` here?

I wonder if there would be another way for this. [In Django documentation](https://docs.djangoproject.com/en/4.2/ref/models/fields/#null) using `null=True` on `CharField` and `TextField` is discouraged. By allowing this field to be `NULL` there's now two different values for "no data" `""` and `NULL`.

And I see you've refactored other Char/TextFields as well to take in null, but I wonder if `blank=True, default=""` would also work? Can we use that instead of `null=True, default=None`?

Reply:
The reason for `null=True` was to support {"name": null} in payload as you expected.

We can send empty string from the UI as well to get rid of `null=True, default=None` or remove `name` from the payload. Changed as suggested

Reply:
Discussed this a bit with @charn. After studying the already existing models and API, it unfortunately seems that `null=True` is very dominant among the `CharField`s and `TextField`s in LE. Therefore it is actually probably the less bad option to make these registration fields nullable as well to keep things consistent. 😅

Reply:
Ok 😅. Changed SignUp model fields nullable.